src/System.ComponentModel.TypeConverter/src/System/ComponentModel/DateTimeConverter.cs

```diff
@@ -118,6 +121,37 @@ public override object ConvertTo(ITypeDescriptorContext context, CultureInfo cul
                 return dt.ToString(format, CultureInfo.CurrentCulture);
             }
 
+            if (destinationType == typeof(InstanceDescriptor) && value is DateTime) 
+            {
+                DateTime dt = (DateTime)value;
+                
+                if (dt.Ticks == 0) 
+                {
+                    // Special case for empty DateTime
+                    ConstructorInfo ctr = typeof(DateTime).GetConstructor(new Type[] { typeof(Int64) });
```

Comment:
How common is this?  If not common, fine.  If common, seems like we should cache `ctr`.  Same goes for `ctor` below.